app/models/suggestion.py

```diff
@@ -34,11 +39,15 @@ def generate_fake_inserts(count=20):
             s_read = 1
             s_timestamp = datetime.now(pytz.timezone('US/Eastern'))
             s_contact_name = fake.word()
-            s_contact_email = fake.word() + "@" + fake.word() + ".com"
-            s_contact_number = "123-456-7890"
-            s_insert = Suggestion(suggestion_text=s_text,
-                                  read=s_read, submission_time=s_timestamp, contact_name=s_contact_name,
-                                  contact_email=s_contact_email, contact_phone_number=s_contact_number)
+            s_contact_email = fake.word() + '@' + fake.word() + '.com'
```

Comment:
Ah I see. I was planning to make it optional, but we can remove it if AC would prefer not to have it.